commit d506cd2dbe14a9f9901112e8d16b5b7f564404f7
Author: Matías Fonzo <selk@dragora.org>
Date:   Sat Jan 25 19:05:53 2020 -0300

    Redirect some headers noticed by the compiler
    
    [..]
    In file included from /usr/src/qi/build/tdebase-trinity-14.0.7/twin/compton-tde/common.h:102,
                     from /usr/src/qi/build/tdebase-trinity-14.0.7/twin/compton-tde/compton.h:10,
                     from /usr/src/qi/build/tdebase-trinity-14.0.7/twin/compton-tde/compton.c:12:
    /usr/include/sys/poll.h:1:2: warning: #warning redirecting incorrect #include <sys/poll.h> to <poll.h> [-Wcpp]
        1 | #warning redirecting incorrect #include <sys/poll.h> to <poll.h>
    [..]
    
    For reference: https://mirror.git.trinitydesktop.org/gitea/TDE/tdelibs/pulls/64
    
    Signed-off-by: Matías Fonzo <selk@dragora.org>

diff --git a/twin/compton-tde/common.h b/twin/compton-tde/common.h
index bcc2ad68..9091fc58 100644
--- a/twin/compton-tde/common.h
+++ b/twin/compton-tde/common.h
@@ -99,7 +99,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <inttypes.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <assert.h>
 #include <time.h>
 #include <ctype.h>
